glob.go

```diff
@@ -98,6 +100,39 @@ func doGlob(fsys fs.FS, pattern string, m []string, firstSegment bool) (matches
 	return
 }
 
+// FilepathGlob returns the names of all files matching pattern or nil if there is
+// no matching file.
+// The syntax of pattern is the same as in Match(). The pattern
+// may describe hierarchical names such as usr/*/bin/ed.
+//
+// FilepathGlob ignores file system errors such as I/O errors reading directories.
+// The only possible returned error is ErrBadPattern, reporting that the
+// pattern is malformed.
+//
+// Note: this is meant as a drop-in replacement for path/filepath.Glob().
+// The matching paths have the same format as what is returned by path/ilepath.Glob().
+func FilepathGlob(pattern string) ([]string, error) {
+	pattern = filepath.Clean(pattern)
+	base, f := SplitPattern(pattern)
+	if base == "" {
```

Comment:
This is a bug in `SplitPattern` - it should return a base of `"/"` in this case. I'll fix that. Note that it should _not_ be `os.PathSeparator` because `DirFS` always uses `/`, even on Windows.